components/apimgt/org.wso2.carbon.apimgt.rest.api.authenticator/src/main/java/org/wso2/carbon/apimgt/rest/api/authenticator/AuthenticatorService.java

```diff
@@ -77,15 +77,16 @@ public AuthenticatorService(KeyManager keyManager,SystemApplicationDao systemApp
      * @return oAuthData - A JsonObject with DCR application details, scopes, auth endpoint, and SSO is enabled or not
```

Comment:
Shall we update the javadocs by adding the uiServiceURL as well?

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/4843/commits/2b8e1c5984040d3434b1ec316bae741424463400